### ISSUE TITLE
Infinite spinning circle

### ISSUE THREAD

**Issue by NateNate60** [shesek/spark-wallet#153]
Similar symptoms to #143, but the install is not a fresh install. I installed the spark-wallet package on my Ubuntu 20.04 server running bog-standard Bitcoin Core and C-Lightning.

`~/.spark-wallet/config contains the following`:

```
ln-path=/home/nate/.lightning
login=[username]:[password]
port=9737
```

When running, the program DOES seem to be giving 200: OK, but in the browser and Android app, it never progresses past the spinning tricolour circle.

```
HTTPS server running on https://0.0.0.0:9737
POST /rpc 200 3.641 ms - 581
POST /rpc 200 13.721 ms - 317
POST /rpc 200 26.136 ms - -
POST /rpc 200 3.507 ms - -
POST /rpc 200 3.337 ms - -
GET /worker.js 304 6.713 ms - -
POST /rpc 200 4.188 ms - -
POST /rpc 200 5.350 ms - -
POST /rpc 200 3.059 ms - 581
POST /rpc 200 6.034 ms - 317
POST /rpc 200 3.119 ms - -
POST /rpc 200 2.425 ms - 581
POST /rpc 200 3.731 ms - -
POST /rpc 200 3.385 ms - -
POST /rpc 200 7.714 ms - 317
POST /rpc 200 2.889 ms - 581
GET /stream?access-key=[         ACCESS KEY         ] 200 11.112 ms - -
^C
nate@nates-server:~$
```

**Comment by shesek** [comment]
Can you check the browser developer console (Ctrl-Shift-I) and see if there are any errors there?

If you're seeing `t.amount_msat is undefined` or `Not a proper lightning payment request`, then it was recently fixed and will shortly be released in v0.2.16.

Would be great if you could confirm if its that or something else. :) Thanks!

**Comment by shesek** [comment]
I released v0.2.16, can you check if that fixes your issue?

**Comment by NateNate60** [comment]
I haven't touched anything from yesterday, but it now appears to be doing something else:
![tempsnip65](https://user-images.githubusercontent.com/41277019/89927567-5a629980-dbbb-11ea-8f30-c302f7f03374.png)

I will go install v0.2.16 and see if the issue is fixed and update again.

**Comment by NateNate60** [comment]
It appears to still be happening

![tempsnip65](https://user-images.githubusercontent.com/41277019/89929360-00af9e80-dbbe-11ea-824d-ced8ce39f6bb.png)

**Comment by shesek** [comment]
Can you try doing a hard refresh with Ctrl-Shift-R and see if that changes anything? Your browser might still have the v0.2.15 assets cached.

**Comment by NateNate60** [comment]
This works, but my Android phone appears to still be having this problem, even after a re-install from Google Play. I solved this by manually installing through the APK file provided in this repository's Releases, and it seems to work now. What that probably means is that the updated package has not been pushed to the Play Store yet, or the update is not yet live. I also had to manually re-install it on my Windows PC.

So for those stumbling on this thread in the future, the solution is to re-install everything and force-reload your browsers.
